packages/docs/src/examples/border-radius/misc-pill-and-circle.vue

```diff
@@ -1,24 +1,11 @@
 <template>
   <v-row class="text-center" justify="center">
-    <v-col
-      cols="12"
-      md="3"
-    >
-      <div
-        class="pa-4 secondary text-no-wrap rounded-pill"
-      >
-        .rounded-pill
-      </div>
+    <v-col cols="12" md="3">
+      <div class="pa-4 secondary text-no-wrap rounded-pill">.rounded-pill</div>
     </v-col>
 
-    <v-col
-      cols="12"
-      md="3"
-    >
-      <div
-        class="pa-7 secondary rounded-circle d-inline-block"
-      ></div>
-
+    <v-col cols="12" md="3">
+      <div class="pa-7 secondary rounded-circle d-inline-block"></div>
```

Comment:
this is missing content

Reply:
![image](https://user-images.githubusercontent.com/42810389/160270037-5f5f42e9-bc3e-406e-9da5-33e37b277a7e.png)
Because the is the expected output